ovn-tester/ovn_tester.py

```diff
@@ -304,25 +327,39 @@ def run_test_netpol_multitenant(ovn, global_cfg, cfg):
         for ns in all_ns:
             ns.unprovision()
 
+def cluster_density_run(ovn, burst=1):
+    ns = [
+        Namespace(ovn, 'NS_'.join(random.choice(string.ascii_lowercase)))
+        for _ in range(burst)
+    ]
+    # create 6 short lived "build" pods
+    build_ports = ovn.provision_ports(6*burst)
```

Comment:
In the "startup" stage we don't need these 6 ports, we just remove them at the end of the function.